src/components/minionExecute.jsx

```diff
@@ -110,7 +109,12 @@ const MinionExecute = ({
 
     if (hideMinionExecuteButton === false) {
       return (
-        <Button onClick={handleExecute} disabled={!isCorrectChain}>
+        <Button
+          onClick={handleExecute}
+          disabled={
+            !isCorrectChain || (minionAction?.memberOnlyEnabled && !isMember)
+          }
+        >
```

Comment:
I think I see the problem. Line 82 `if (proposalType === PROPOSAL_TYPES.MINION_BUYOUT)` makes this change only apply there. It needs to be on all minion proposals.

Reply:
Nevermind, I think this didn't get merged right.